src/pages/requests/index.tsx

```diff
@@ -55,6 +55,15 @@ export function Requests() {
       });
     }
 
+    if (locationDict) {
+      addFilter({
+        filterName: "Cities",
+        filterItems: Object.values(locationDict).map((location): FilterItem => ({ id: location.name.toLowerCase().replace(" ", "-"), selected: false, text: location.name })),
```

Comment:
can we use "location.id" as id?

Reply:
So I see the locations API is returning a number for the `id` (`1`, `2`, `3`, etc.) , where the supplies API is returning a string (e.g., `baby_food`, `water`, etc.).

I think we should be consistent here and return the same type for `id`.

Reply:
It's not that we own the datasource that we can change it.
Few weeks ago when the whole thing was starting I did try to convince the team to have numeric ids for both.
It did work for cities, but for categories they didn't listen to me.

So for the time being we have to work with what we have.

Reply:
Thanks for the background! Would you mind if this is fixed in a follow-up PR? Changing it to `location.id`, will also require changing the filter ID type to `string | number`, which creates a cascade of changes (other areas in the code where `string` is expected).

Reply:
@MaciejReimann don't see a problem with debugging and integers